pkg/appconsts/global_consts.go

```diff
@@ -13,55 +11,9 @@ import (
 //
 // They can not change throughout the lifetime of a network.
 const (
-	// NamespaceVersionSize is the size of a namespace version in bytes.
-	NamespaceVersionSize = ns.NamespaceVersionSize
-	// NamespaceVersionMaxValue is the maximum value a namespace version can be.
-	// This const must be updated if NamespaceVersionSize is changed.
-	NamespaceVersionMaxValue = math.MaxUint8
-
-	// NamespaceIDSize is the size of a namespace ID in bytes.
-	NamespaceIDSize = ns.NamespaceIDSize
-
-	// NamespaceSize is the size of a namespace (version + ID) in bytes.
-	NamespaceSize = ns.NamespaceSize
-
-	// ShareSize is the size of a share in bytes.
-	ShareSize = 512
-
-	// ShareInfoBytes is the number of bytes reserved for information. The info
-	// byte contains the share version and a sequence start indicator.
-	ShareInfoBytes = 1
-
-	// SequenceLenBytes is the number of bytes reserved for the sequence length
-	// that is present in the first share of a sequence.
-	SequenceLenBytes = 4
-
-	// ShareVersionZero is the first share version format.
-	ShareVersionZero = uint8(0)
-
 	// DefaultShareVersion is the defacto share version. Use this if you are
 	// unsure of which version to use.
-	DefaultShareVersion = ShareVersionZero
-
-	// CompactShareReservedBytes is the number of bytes reserved for the location of
-	// the first unit (transaction, ISR) in a compact share.
-	CompactShareReservedBytes = 4
-
-	// FirstCompactShareContentSize is the number of bytes usable for data in
-	// the first compact share of a sequence.
-	FirstCompactShareContentSize = ShareSize - NamespaceSize - ShareInfoBytes - SequenceLenBytes - CompactShareReservedBytes
-
-	// ContinuationCompactShareContentSize is the number of bytes usable for
-	// data in a continuation compact share of a sequence.
-	ContinuationCompactShareContentSize = ShareSize - NamespaceSize - ShareInfoBytes - CompactShareReservedBytes
-
-	// FirstSparseShareContentSize is the number of bytes usable for data in the
-	// first sparse share of a sequence.
-	FirstSparseShareContentSize = ShareSize - NamespaceSize - ShareInfoBytes - SequenceLenBytes
-
-	// ContinuationSparseShareContentSize is the number of bytes usable for data
-	// in a continuation sparse share of a sequence.
-	ContinuationSparseShareContentSize = ShareSize - NamespaceSize - ShareInfoBytes
+	DefaultShareVersion = share.ShareVersionZero
```

Comment:
I actually don't know what is meant by the default here. I have left it as is because v1 is a new feature and I a) don't know if many people will use it and b) should be opt in anyway.

Maybe if this is just for tests we can change it but it would require a large refactor since the new version requires new arguments